bms_current_test.py

```diff
@@ -0,0 +1,16 @@
+import math
+import unittest
+import bms_current_ranges
+
+class test_empty_current_ranges(unittest.TestCase):
+  def test_failing_current_ranges(self):
+    current_input = []
+    self.assertTrue(bms_current_ranges.check_empty_input(current_input) == "Empty_Input")
+    
+  def test_is_nan_current_ranges(self):
```

Comment:
Use the test name to express functionality. Like `test_nan_in_the_input_is_reported_as_invalid`
Ensure the input and output figure in the test name.
You can use long names because we don't need to write code to call this.